.github/workflows/ci.yml

```diff
@@ -275,9 +275,11 @@ jobs:
             ~/.cargo/git/db/
             target/
           key: codecov-cargo-${{ hashFiles('**/Cargo.lock') }}
+      - name: Install Rust toolchain
+        run: rustup toolchain install 1.71.0-x86_64-unknown-linux-gnu
       - name: Generate coverage report
         run: >
-          cargo tarpaulin
+          cargo +1.71.0 tarpaulin
```

Comment:
can you explain the changes to this file?

Reply:
It seems that GitHub action runners have been updated and no longer have 1.71.0 toolchain by default. So we need to install it manually. But here specifically we probably don't need it because cargo will pick it up from toolchain.toml